src/js/navigation.js

```diff
@@ -82,8 +82,18 @@ document.addEventListener('DOMContentLoaded', () => {
         });
 
         // Fetch menu items array
+
+        const baseUrl = process.env.NODE_ENV === 'development'
+            ? 'http://localhost:3000'
+            : '';
+
+        const url = process.env.NODE_ENV === 'development'
```

Comment:
Maak je `getApiUrl` functie herbruikbaar in een apart bestand `utils.js` bijvoorbeeld. Dan kan je hem hier gebruiken.